lib/parse-args.js

```diff
@@ -13,7 +13,10 @@ function buildYargs (withCommands = false) {
     .options('config', {
       alias: 'c',
       config: true,
-      describe: 'path to JSON configuration file',
+      describe: 'path to JSON configuration file. By default, c8 searches for ' +
+        'files named .c8rc, .c8rc.json, .nycrc, or .nycrc.json, ' +
+        'starting from cwd and walking up the filesystem tree.',
+      defaultDescription: 'as described',
```

Comment:
could this just be `cwd`, and drop the last sentence:

> starting from cwd and walking up the filesystem tree?

Also, rather than writing out `.c8rc`, `.c8rc.json`, etc.,

Why not pull `['.c8rc', '.c8rc.json', '.nycrc', '.nycrc.json']` into a variable, and use the array we pass to `find-up` to create the description? That way someone doesn't need to update in both places.